src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexInterpreter.cs

```diff
@@ -337,6 +337,7 @@ protected override void Go()
             SetOperator(_code.Codes[0]);
             _codepos = 0;
             int advance = -1;
+            ReadOnlySpan<char> runtextSpan = runtext.AsSpan();
```

Comment:
Nit: this can just be:
```suggestion
            ReadOnlySpan<char> runtextSpan = runtext;
```